website/notifications/views.py

```diff
@@ -0,0 +1,72 @@
+import httplib as http
```

Comment:
NP: Imports should be grouped into system, third-party, and local source tree each of which are separated by a single empty line.